truffleruby/src/main/ruby/core/rbconfig.rb

```diff
@@ -148,6 +144,11 @@ def RbConfig.expand(val, config = CONFIG)
     val.replace(newval) unless newval == val
     val
   end
+
+  CONFIG.each_value do |val|
+    RbConfig::expand(val)
+  end
```

Comment:
This mean we'll run a fair amount of code just to set these values.
I don't think it's worth it, and it does not enhance readability IMHO.
If we want to reuse parts, how about just using String interpolation?